draft-law-moq-warpstreamingformat.md

```diff
@@ -87,70 +91,20 @@ Media tracks SHOULD be media-time aligned. CMAF {{CMAF}} Aligned Switching Sets
-
-~~~
-Track Change Descriptor {
-  full track name length (i),
-  full track name (..),
-  operation (1),
-  change payload(..)
-}
-~~~
-{: #warpmedia-track-descriptor title="Track change descriptor"}
-* Full track name length: the length of the full track name field
-* Full track name: the Full Track Name as defined by {{MoQTransport}} (Sect 2.3.1). Track names MUST never be reused. If a track is published and then unpublished, it must be allocated a new track name before it is re-published. A catalog MUST NOT reference itself i.e the the track name must not be "catalog".
-* Operation: a binary flag. 1 if the track is being added and 0 if it is being deleted. A publisher MUST NOT signal deletion of a track that has not been previously added.
-* Change payload:  depends upon the value of the operation flag. If the operation is a 1 (add), then it SHALL hold an Initialization Header. If the operation is 0 (delete), then it SHALL hold a Deletion Header.
-
-~~~
-Initialization Header {
-  init length (i)
-  init payload (..)
-}
-~~~
-{: #warpmedia-initialization-header title="Initialization Header"}
-* Init length: the length of the init payload
-* Init payload:
-The init payload MUST consist of a File Type Box (ftyp) followed by a Movie Box (moov). This Movie Box (moov) consists of Movie Header Boxes (mvhd), Track Header Boxes (tkhd), Track Boxes (trak), followed by a final Movie Extends Box (mvex). These boxes MUST NOT contain any samples and MUST have a duration of zero. A Common Media Application Format Header {{CMAF}} meets all these requirements.
-
-~~~
-Deletion Header {
-  Last group: (i),
-  Last object: (i)
-}
-~~~
-{: #warpmedia-deletion-header title="Deletion Header"}
-* Last group: holds the last {{MoQTransport}} Group sequence number published under that track name.
-* Last object: holds the last {{MoQTransport}} Object sequence number published under that track name.
+# Catalog
+
+WARP uses the Common Catalog Format {[COMMON-CATALOG-FORMAT}} to describe the content being produced by a publisher.
+
+Per Sect 5.1 of {{COMMON-CATALOG-FORMAT}}, WARP registers an entry in the "MoQ Streaming Format Type" table.  The type value is 0x001, the name is "WARP Streaming Format" and the RFC is XXX.
+
+Every WARP catalog MUST declare a streaming format type (See Sect 3.2.1 of {{COMMON-CATALOG-FORMAT}}) value of 1.
+
+Every WARP catalog MUST declare a streaming format version (See Sect 3.2.1 of {{COMMON-CATALOG-FORMAT}}) corresponding to the version of this document.
+
+The catalog track MUST have a track name of "catalog". A catalog object MAY be independent of other catalog objects or it MAY represent a delta update of a prior catalog object. The first catalog object published within a new group MUST be independent.  A catalog object SHOULD only be published only when the availability of tracks changes.
+
+Each catalog update MUST be mapped to a discreet moq-transport object.
+
```

Comment:
we should also add at group boundaries the catalog object MUST BE non-delta update.

Reply:
The existing language "The first catalog object published within a new group MUST be independent" already adds this requirement.